docs/nv2-sign-verify-aks.md

```diff
@@ -191,227 +46,79 @@ In this step, Gatekeeper will be configured, enabling deployment policies.
-        --set ratifyTestCert="$PUBLIC_KEY"
+The constraint template applied denies any pods which are not signed in the `demo` namespace.
 
-    kubectl apply -f ./ratify/charts/ratify-gatekeeper/templates/constraint.yaml
+## Apply policy and deploy images
 
-    cat ./ratify/charts/ratify-gatekeeper/templates/constraint.yaml
-    ```
+Now that gatekeeper and ratify are installed, a K8sSignedImages gatekeeper constraint policy needs to be applied to enforce only allowing signed images to be deployed.  After applying this policy we will show a signed imaged deployed successfully and a non-signed or trusted image deployed with a failure.
 
-4. Deploy `hello-world:latest` to show a functioning cluster
+1.  Apply a constraint policy to only allow signed images
 
-    ```bash
-    kubectl run hello-world \
-      --image=mcr.microsoft.com/azuredocs/aci-helloworld:latest \
-      -n demo
-    ```
+```azurecli-interactive
+kubectl apply -f https://deislabs.github.io/ratify/charts/ratify-gatekeeper/templates/constraint.yaml
+```
 
-    > Note: due to [Ratify issue #92](https://github.com/deislabs/ratify/issues/92), this may pass validation.
+The constraint template applied denies any pods which are not signed in the `demo` namespace.
 
-5. List the running pods
+2. Create the demo namespace on the AKS cluster
 
     ```bash
-    kubectl get pods -A
-    ```
-
-## Build an Image
-
-1. Build and Push a new image with ACR Tasks
-
-    ```azure-cli
-    az acr build -r $ACR_NAME -t $IMAGE $IMAGE_SOURCE
+    kubectl create ns demo
     ```
 
-2. Sign the container image
+3. Deploy a signed image to AKS - with a success
 
     ```bash
-    notation sign --key $KEY_NAME $IMAGE 
+    kubectl run hello-world --image=mcr.microsoft.com/azuredocs/aci-helloworld:latest -n demo
+    kubectl get pods -n demo -w
     ```
 
-3. Deploy the signed `net-monitor:v1` image
+4. Deploy an unsigned image to AKS - with a failure
 
     ```bash
-    kubectl run net-monitor --image=$IMAGE -n demo
+    kubectl run hello-world \
+      --image=mcr.microsoft.com/azuredocs/aci-helloworld:latest \
+      -n demo
     ```
 
-4. List the running pods
+5. List the running pods
 
     ```bash
     kubectl get pods -A
     ```
 
-5. Delete the pod
-
-    ```bash
-    kubectl delete pod net-monitor -n demo
-    ```
-
 ## Reset
 
-The following steps cleanup the resources, resetting to an AKS with gatekeeper state.
-
-1. Remove keys, certificates and notation `config.json`
-
-    ```bash
-    rm ~/.config/notation/config.json
-    ```
-
-1. Remove the Plugin
-
-    ```bash
-    az keyvault certificate delete -n $KEY_NAME --vault-name $AKV_NAME
-    az keyvault certificate purge --vault-name $AKV_NAME --name $KEY_NAME
-    ```
-
-2. Clear the ACR repo
-
-    ```bash
-    az acr repository delete \
-        -n $ACR_NAME \
-        --repository $ACR_REPO -y
-    ```
-
-3. Clear up AKS resources, leaving AKS in place
+Clear up ratify and gatekeeper resources, leaving AKS in place
```

Comment:
Currently this is missing pulling from ACR. I'm guessing you will add this and also leave the registry in place?

Reply:
Yes, the thinking is they will use the images they already pulled from the previous article.